src/styles/sidebar/components/annotation-publish-control.scss

```diff
@@ -39,18 +74,21 @@
     margin-left: 8px;
 
     border: none;
-    background-color: transparent;
+    background-color: $color-dove-gray;
```

Comment:
Setting background color explicitly here and in the non-hover rule below is necessary to get the "split" hover effect between the primary action and the drop-down menu arrow.

I was concerned that the `transparent` here before was to avoid conflicting with any applied branding-theming styles, but tested and saw that the rule specificity of the inline styles means that no hover effect gets applied anyway—translation: custom branding applies still without interference, and if you have custom branding styles, you don't get rollover at present anyway.

This appears to work as I expect. Let me know if I'm overlooking something.